### DISCUSSION TITLE
質問の回答のコメントでMarkdownを使用出来るようにしてほしい

### DISCUSSION THREAD

**Discussion by radian-jp** [increments/qiita-discussions#363] [要望・アイデア・不具合]
記事のコメントにはMarkdownが使えるのですが、質問の回答のコメントでは使用出来ないので、こちらでも使えると便利になると思うのですが、いかがでしょうか？

**Comment by tanaka-asami415** [comment]
@radian-jp 
ご意見ありがとうございます！
質問機能のアップデートの際に、実施予定ですのでお待ちいただけると幸いです💪

**Comment by radian-jp** [comment]
いつの間にかMarkdown使用出来るようになっていますね！対応ありがとうございます！
一点気になったのが、過去のコメントの改行が無くなって表示されます。

![名称未定-1](https://user-images.githubusercontent.com/12453432/232261191-5093909b-c8d7-426f-b6f1-423969315a5d.png)

編集すると改行されているので、改行情報が欠落したという訳ではなさそうです。

![名称未定-2](https://user-images.githubusercontent.com/12453432/232261201-25dabc01-3953-42b0-9284-63a52916dd37.png)

この状態で更新すると、改行は反映されて表示されました。

![名称未定-3](https://user-images.githubusercontent.com/12453432/232261240-cc9893bc-5f03-4832-a8d6-d726331942d6.png)

**Reply by kimkim0814** [reply]
@radian-jp
ご報告ができておらず申し訳ありません🙏コメントでもMarkdownが使えるようになりました！
https://qiita.com/release-notes/2023-04-14

Markdownのご利用と不具合のご報告いただきありがとうございます🙇
上記について調査したところ、コメントのMarkdown化以前のデータが適切なコンポーネントで使用されておりませんでした。

ご不便をおかけして申し訳ありません。こちら修正いたします。

**Reply by radian-jp** [reply]
今確認したら、正しく改行されるようになっていました。
迅速な修正ありがとうございます。

**Reply by kimkim0814** [reply]
@radian-jp
こちらこそご確認いただきありがとうございます！
過去の回答コメントでMarkdownが改行されない不具合について修正いたしました。ご不便おかけ致しました🙇

引き続きQiitaをよろしくお願いいたします😊
